newdle/api.py

```diff
@@ -96,3 +96,13 @@ def newdle(title, duration, timezone, time_slots, participants):
     db.session.add(newdle)
     db.session.commit()
     return NewdleSchema().jsonify(newdle)
+
+
+@api.route('/newdle/<code>')
+def get_newdle(code):
+    newdle = Newdle.query.filter(Newdle.code == code).first()
```

Comment:
`.filter_by(code=code)`

Reply:
`.one_or_404()`?

Reply:
I didn't realize `.one_or_404()` comes from `flask-sqlalchemy`. I thought it was an Indico thing. Thanks!

Reply:
There's also `first_or_404`, which is fine here since the column is unique